settings.gradle

```diff
@@ -32,6 +32,7 @@ final def module = { final String name, final String path ->
     project(":$name").projectDir = new File("$rootDir/$path")
 }
 
+module 'checker',             'tools/checker'
```

Comment:
As long as we also have `model-verifier` (and thus "checker" and "verifier" may be confused with each other), this probably should be `errorprone-checks`.